<launch>
<node
    pkg="turtlesim"
    type="turtlesim_node"
    name="turtlesim"
/>
<node
    pkg="turtlesim"
    type="turtle_teleop_key"
    name="teleop_key"
    required="true"
    launch-prefix="xterm -e"
/>
<node
    pkg="beginner_tutorials"
    type="move_second_turtle"
    name="pregunta1"
    output="screen"
/>
</launch>